backend/server.js

```diff
@@ -2,38 +2,32 @@ if (process.env.NODE_ENV !== 'production') { // can load .env file while server
     require('dotenv').config();
 }
 
-const authHelper = require('./authHelper')
-const prisma = require('./database/db')
-const express = require('express') 
-const app = express()
-const port = 8080
+const express = require('express'); 
+const app = express();
+const cors = require('cors');
 const bcrypt = require('bcrypt')
-const session = require('express-session')
-const {PrismaSessionStore} = require('@quixo3/prisma-session-store')
-const {PrismaClient} = require('@prisma/client')
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+const port = 8080;
+const {authenticateSession, userSession, userSessionRouter} = require('./routes/userSession.js');
 const usernameRouter = require('./routes/usernameRoute');
-app.use(cors({ origin:'http://localhost:3000'}));
-app.use('/reset-username-form', usernameRouter)
+const authHelper = require('./authHelper')
+import {createSession} from './handlers/createSession.js';
```

Comment:
can you help me understand what this is doing?

Reply:
actually doesn't this line throw an error if you run the server? import is not a recognized keyword in the backend without additional set up.

Reply:
im not sure what you mean. are you talking about the lines in the green or red?